lib/chef/provider/package.rb

```diff
@@ -262,6 +230,11 @@ def package_locked(name, version)
         raise Chef::Exceptions::UnsupportedAction, "#{self} has no way to detect if package is locked"
       end
 
+      # Subclasses will override this to a method and provide a preseed file path
+      # only apt_package and dpkg_package are overriding this method
```

Comment:
Would recommend removing hte reference to apt_package/dpkg_package - this is the kind of thing that changes over time, and this is the kind of comment that we forget to update because it's not in the path of the change.